src/core/objects/object_mover.h

```diff
@@ -64,16 +73,24 @@ class ObjectMover
 	/** Adds a text handle to the set of elements to move. */
 	void addTextHandle(TextObject* text, MapCoordVector::size_type handle);
 	
+	enum HandleOpMode {
+		Never,
+		Click,
+		Always
+	};
+
```

Comment:
I would prefer to have the enum at the beginning of the class definition.

Reply:
I think that this enum and its members need documentation.

Reply:
Indeed. The `enum` placement is somewhat random. Will fix.